### PR TITLE
deal with one input geno

### PR DESCRIPTION
**Pull request by changebio**

### CODE REVIEW

File: GWAS/Region_Extraction.ipynb [hunk 1]
```diff
@@ -151,25 +151,25 @@
     "# Work directory where output will be saved to\n",
     "parameter: cwd = path\n",
     "# Region specifications\n",
-    "parameter: region_file = path\n",
+    "parameter: region_file = path()\n",
```
Comment (gaow) [inline GWAS/Region_Extraction.ipynb:154]: When using `path` and not `path()`, we require that the input must be provided. is there a specific reason that you have to use `path()` instead?

**Comment by changebio** [comment]
Please merge the PR.

**Comment by gaow** [comment]
@changebio am not going to merge this until you address to my comment above.

**Comment by changebio** [comment]
it has been updated to `path`, instead of `path()`.

**Comment by gaow** [comment]
@changebio Thanks. have you pushed it? I don't see the change

**Comment by changebio** [comment]
Yes, Have you seen it? it was in my last push. (https://github.com/cumc/bioworkflows/pull/153/commits/27b0f1d44c6c6372fada5a0e262ad3f64793dca3)

File: GWAS/liftover.ipynb [hunk 2]
```diff
@@ -251,13 +251,17 @@
     "output: f'{cwd}/{output_file}'\n",
     "python: input = f'{cwd:a}/utils.py', expand = '${ }', stderr = f'{_output[0]:n}.stderr', stdout = f'{_output[0]:n}.stdout'\n",
     "    \n",
+    "    \n",
+    "    import os.path\n",
     "    input_path=${_input[0]:r}\n",
     "    output_path=${_output[0]:r}\n",
-    "    fr = '${fr}'\n",
-    "    to = '${to}'\n",
+    "    fr = f'${fr}'\n",
```
Comment (gaow) [inline GWAS/liftover.ipynb:258]: please revert these lines we dont need f-string here

File: GWAS/liftover.ipynb [hunk 3]
```diff
@@ -251,13 +251,17 @@
...
+    "    to = f'${to}'\n",
     "    remove_missing=${remove_missing}\n",
     "    rename = ${rename}\n",
-    "    yml_file = '${yml_file}'\n",
+    "    yml_file = f'${yml_file}'\n",
```
Comment (gaow) [inline GWAS/liftover.ipynb:262]: you can try this: 

```
import pathlib
yml_file  = pathlib.Path(${yml_file:r})
```
this allows you to use `is_file()` later. of course you can do it with other approaches.

**Pull request merged**